pages/trouw-opties/[slug].tsx

```diff
@@ -117,60 +167,39 @@ const BlogPost: NextPage = () => {
-                        <RadioButton2
-                          novalidate={true}
-                          id={event.id}
-                          name="event"
-                          value={event.id}
-                          required
-                          onChange={onChangeDateHandler}
-                        />
-                        <FormLabel htmlFor={event.id} type="radio">
-                          <span aria-label="negen uur tot tien over negen">
-                            <TimeValue dateTime={event.startDateTime} locale={locale} />
-                            {" – "}
-                            <TimeValue dateTime={event.endDateTime} locale={locale} />
-                            {" uur"}
-                          </span>
-                        </FormLabel>
-                      </FormField>
-                    ))}
-                  </Fieldset>
-                  <Fieldset>
-                    <FieldsetLegend>Uitgebreid trouwen — Zelf de plaats bepalen</FieldsetLegend>
-                    {getEvents("uitgebreid-huwelijk", selectedDate).map((event) => (
-                      <FormField key={event.id}>
-                        <RadioButton2
-                          novalidate={true}
-                          id={event.id}
-                          name="event"
-                          value={event.id}
-                          required
-                          onChange={onChangeDateHandler}
-                        />
-                        <FormLabel htmlFor={event.id}>
-                          <span aria-label="negen uur tot tien over negen">
-                            <TimeValue dateTime={event.startDateTime} locale={locale} />
-                            {" – "}
-                            <TimeValue dateTime={event.endDateTime} locale={locale} />
-                            {" uur"}
-                          </span>
-                        </FormLabel>
-                      </FormField>
-                    ))}
-                  </Fieldset>
-
-                  <ButtonGroup>
-                    <Button type="submit" appearance="primary-action-button">
-                      Ja, dit wil ik!
-                    </Button>
-                  </ButtonGroup>
+                  {calendarData.selectedDate && (
+                    <div>
+                      <p>
+                        <DateValue dateTime={calendarData.selectedDate.toISOString()} locale={locale} />
+                      </p>
+                      {ceremonies.map((ceremony, idx) => (
+                        <Fieldset key={idx}>
+                          <FieldsetLegend>{ceremony.type}</FieldsetLegend>
+                          {calendarData.selectedDate &&
+                            availabilities[format(calendarData.selectedDate, dateFormat)]
+                              ?.filter((slot) => slot.resources.includes(ceremony.id))
+                              .map((slot, idx) => (
```

Comment:
I think you might want to use `role="radiogroup"` and base the HTML for the radio-group on this storybook template of Utrecht https://github.com/nl-design-system/utrecht/blob/main/packages/storybook-react/src/stories/FormFieldRadioGroup.stories.tsx#L103